### ISSUE TITLE
support for require('@scoped/package-name')

### ISSUE THREAD

**Issue by davidknezic** [substack/node-browserify#1494]
Is it possible that browserify doesn't support [scoped packages](https://docs.npmjs.com/misc/scope)?

I'm trying to require a package that has a scope in its name:

```js
require('@axa-ch/style-guide')
```

However, this fails with *cannot find module '@axa-ch/style-guide'*.

Also, importing the package by path didn't work neither...

```js
require('../node_modules/@axa-ch/style-guide')
```

... while it would work for other modules...

```js
require('../node_modules/baconjs')
```

**Comment by bjoerge** [comment]
I've never had any issues with requiring scoped packages. I'm pretty sure they should just work. Can you publish a repro case?

**Comment by davidknezic** [comment]
You're totally right @bjoerge! Everything is fine with scoped module loading...

While I was reproducing the case, I found out that my module simply missed the `{ main: 'js/index.js' }` file. Since it didn't find the main file, browserify returned with the error *cannot find module '@axa-ch/style-guide'*.

While the error message could've been more precise, this is absolutely my fault. Closing. Thx @bjoerge!
